app/models/bot.rb

```diff
@@ -11,4 +11,10 @@ def eligible_collaborators
         .order(:username)
         .map{ |u| [u.username, u.id] }
   end
+
+  def active_instance
+    BotInstance.where(bot: self)
+               .where('last_ping > ?', 3.minutes.ago)
```

Comment:
Probably should either take the instance with the most recent seen time, or rename this to `active_instances` and return an array. With Smokey, for example, we sometimes run two active instances for testing or whatnot.